sklearn/utils/multiclass.py

```diff
@@ -431,17 +431,12 @@ def _ovr_decision_function(predictions, confidences, n_classes):
             votes[predictions[:, k] == 1, j] += 1
             k += 1
 
-    max_confidences = sum_of_confidences.max()
-    min_confidences = sum_of_confidences.min()
-
-    if max_confidences == min_confidences:
-        return votes
-
-    # Scale the sum_of_confidences to (-0.5, 0.5) and add it with votes.
+    # Monotonically transform the sum_of_confidences to (-0.5, 0.5)
```

Comment:
Shouldn't it be (-1/3, 1/3)? I would even add here the formula of the monotone transformation that is used and maybe move here the comment that you inserted below L439 explaining why 1/3 is used instead of 1/2:
`# We use 1/3 instead of 1/2 to ensure that we won't reach the limits and produce ties`